tests/README.md

```diff
@@ -2,27 +2,51 @@
 
 ## Overview
 
-This project contains UI smoke tests for Busola. Copy your cluster's kubeconfig file to [`fixtures/kubeconfig.yaml`](fixtures/kubeconfig.yaml).
+This project contains smoke and integration UI tests for Busola. Copy your cluster's kubeconfig file to [`fixtures/kubeconfig.yaml`](fixtures/kubeconfig.yaml).
 
 ## Installation
 
 To install dependencies, run the `npm install` command.
 
 ## Development
 
-To run UI tests inside a container in the cluster using a Chrome browser, use this command:
+### Run Cypress UI tests in the headless mode
+
+To run Cypress UI tests using a Chrome browser in `headless mode`,
+pointing to `remote Busola` cluster with default `local.kyma.dev` domain, use this command:
+
+```bash
+npm test
+```
+
+Pointing to `remote Busola` cluster with `custom` domain, use this command:
```

Comment:
```suggestion
To run the tests, pointing to a `remote Busola` cluster with `custom` domain, use this command:
```

Because ```Pointing to `remote Busola` cluster with `custom` domain, use this command:``` isn't a full/complete sentence, I tried to add what the purpose of the command is. But if it doesn't make sense feel free to replace it. 😄